### ISSUE TITLE
pactVerify is skipped when first Provider has one pact records failed

### ISSUE THREAD

**Issue by minhdoan159** [DiUS/pact-jvm#772]
Hi team, 
I want to run pactVerify (using pact-jvm-provider-gradle 3.5.9) to verify pact records on Broker of two providers.
Command line: gradle pactVerify
Configure in build.gradle:
`serviceProviders {
        FEATURE_PROVIDER {
            //isDependencyForPactVerify = false
            //insecure = true
            requestFilter = { req ->
                  //
                }

            }
         CONVENTION_PROVIDER {
             //isDependencyForPactVerify = false
             //insecure = true
            requestFilter = { req ->
}}}`
But it seems when any first provider's record failed, it willl stop the pactVerify task.
Any configuration to ignore failure and continue remaining tests?
Many thanks for your reading and help!

**Comment by uglyog** [comment]
This is due to the way Gradle works. Each provider is verified in a separate Gradle task, and Gradle will stop the build as soon as any task fails. We can change the behaviour to always run all the provider tasks, but the problem is in this case if the first one fails and the second one passes, the whole build will pass, not fail.

**Comment by minhdoan159** [comment]
Thanks @uglyog so much for your work!
I think in case there are many providers, the result PASSED should mean all of them executed and the detailed result of each should be read carefully.

**Comment by psimms10** [comment]
Yes, the build should fail if any of the PACTs aren't satisfied. I'm seeing this in the JS provider verify.
However, ideally it would be really good to run through all the associated contracts to see the scope of any problem - rather than having to fix the 1st failure, then later find there are other failures that subsequently need fixing.

**Comment by uglyog** [comment]
@psimms10 I would recommend using a JUnit test instead, it will have the behaviour you require
